docs/v0.7/zh/reference/sql/information-schema/columns.md

```diff
@@ -0,0 +1,176 @@
+# COLUMNS
+
+`COLUMNS` 提供了关于表中每列的详细信息。
+
+```sql
+USE INFORMATION_SCHEMA;
+DESC COLUMNS;
+```
+
+结果如下：
+
+```sql
++--------------------------+--------+------+------+---------+---------------+
+| Column                   | Type   | Key  | Null | Default | Semantic Type |
++--------------------------+--------+------+------+---------+---------------+
+| table_catalog            | String |      | NO   |         | FIELD         |
+| table_schema             | String |      | NO   |         | FIELD         |
+| table_name               | String |      | NO   |         | FIELD         |
+| column_name              | String |      | NO   |         | FIELD         |
+| ordinal_position         | Int64  |      | NO   |         | FIELD         |
+| character_maximum_length | Int64  |      | YES  |         | FIELD         |
+| character_octet_length   | Int64  |      | YES  |         | FIELD         |
+| numeric_precision        | Int64  |      | YES  |         | FIELD         |
+| numeric_scale            | Int64  |      | YES  |         | FIELD         |
+| datetime_precision       | Int64  |      | YES  |         | FIELD         |
+| character_set_name       | String |      | YES  |         | FIELD         |
+| collation_name           | String |      | YES  |         | FIELD         |
+| column_key               | String |      | NO   |         | FIELD         |
+| extra                    | String |      | NO   |         | FIELD         |
+| privileges               | String |      | NO   |         | FIELD         |
+| generation_expression    | String |      | NO   |         | FIELD         |
+| greptime_data_type       | String |      | NO   |         | FIELD         |
+| data_type                | String |      | NO   |         | FIELD         |
+| semantic_type            | String |      | NO   |         | FIELD         |
+| column_default           | String |      | YES  |         | FIELD         |
+| is_nullable              | String |      | NO   |         | FIELD         |
+| column_type              | String |      | NO   |         | FIELD         |
+| column_comment           | String |      | YES  |         | FIELD         |
+| srs_id                   | Int64  |      | YES  |         | FIELD         |
++--------------------------+--------+------+------+---------+---------------+
+24 rows in set (0.00 sec)
+```
+
+创建表 `public.t1` 并查询其在 `COLUMNS` 中的信息：
+
+```sql
+CREATE TABLE public.t1 (h STRING, v FLOAT64, ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP() TIME INDEX, PRIMARY KEY(h));
+SELECT * FROM COLUMNS WHERE table_schema='public' AND TABLE_NAME='t1'\G
+```
```

Comment:
````suggestion
```sql
CREATE TABLE public.t1 (h STRING, v FLOAT64, ts TIMESTAMP DEFAULT CURRENT_TIMESTAMP() TIME INDEX, PRIMARY KEY(h));
SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE table_schema='public' AND TABLE_NAME='t1';
```
````

Reply:
Doesn't seems necessary.